Sources/Common/DataModel/KochanekSpline1D/Constants.js

```diff
@@ -0,0 +1,9 @@
+export const boundaryCondition = {
+  DERIVATIVE: 1,
+  SECOND_DERIVATIVE: 2,
+  SECOND_DERIVATIVE_INTERIOR_POINT: 3,
+};
+
+export default {
+  boundaryCondition,
+};
```

Comment:
why not define those constants in Spline1D instead of twice in Cardinal AND Kochanek ?